src/pulse3D/plate_recording.py

```diff
@@ -160,19 +162,16 @@ def __init__(
             self.is_force_data = (
                 "y" in str(_get_excel_metadata_value(self._excel_sheet, TWITCHES_POINT_UP_UUID)).lower()
             )
-
-            self.noise_filter_uuid = (
-                TSP_TO_DEFAULT_FILTER_UUID[self.tissue_sampling_period] if self.is_magnetic_data else None
-            )
-            self.filter_coefficients = (
-                create_filter(self.noise_filter_uuid, self.tissue_sampling_period)
-                if self.noise_filter_uuid
-                else None
+            self.stiffness_factor = None
+            for uuid_ in (PLATEMAP_NAME_UUID, PLATEMAP_LABEL_UUID):
+                self[uuid_] = NOT_APPLICABLE_LABEL
+            # set tissue to force data without ADC or noise filtering
+            adj_raw_tissue_reading = self[TISSUE_SENSOR_READINGS].copy()
+            time_conversion = (
+                MICROSECONDS_PER_CENTIMILLISECOND if self.is_magnetic_data else MICRO_TO_BASE_CONVERSION
```

Comment:
This is only needed for beta 1 data, be careful and intentional when copying/pasting. I think we can skip the time conversion here

Reply:
without the time conversion the pulse3d errors